### ISSUE TITLE
Respect migration scripts

### ISSUE THREAD

**Issue by voronind** [OCA/pylint-odoo#338]
Can we avoid messages for some special cases for migration scripts?

```python
def migrate(cr, version=None):
    pass
```
- [ ] Not camel-case file name
- [ ] Too short `cr` variable name
- [ ] Not used `version` variable name
```
addon/migrations/11.0.2/pre-migration.py:1:0: C0103: Module name "pre-migration" doesn't conform to snake_case naming style (invalid-name)
addon/migrations/11.0.2/pre-migration.py:1:0: C0103: Argument name "cr" doesn't conform to snake_case naming style (invalid-name)
addon/migrations/11.0.2/pre-migration.py:1:16: W0613: Unused argument 'version' (unused-argument)
```

**Comment by voronind** [comment]
I know that I can use config
```ini
[BASIC]
good-names=cr
```
but it's better to have concrete rule.

**Comment by moylop260** [comment]
They are pylint built-in checks no pylint-odoo ones

But you can parameter the regex for each one

e.g. for module name you can consider `pre-` and `post-` in the regex as:
 - `module-rgx=(([a-z_][a-z0-9_]*)|([A-Z][a-zA-Z0-9]+)|(pre\-.*|post\-.*))$`

**Comment by voronind** [comment]
I know that it's pylint checks. But we can add bypasses to pylint-odoo like for `__manifest__.py`

**Comment by moylop260** [comment]
Yes

The exception is supported by pylint itself
using the regex above in your configuration file

Could you make a new PR for considering it in the pylint configuration file of example, please?

  - https://github.com/OCA/pylint-odoo/blob/master/pylint_odoo/examples/.pylintrc

**Comment by voronind** [comment]
But I want to have `pre-...` module in `migration` directory not everywhere.

**Comment by moylop260** [comment]
You have a good point

**Comment by moylop260** [comment]
I just realized that running `pylint` over a global module the migrations script are not checked since that there are not `__init__.py` files import them

But it is a new issue related with `pre-commit` fashion since that it calls all the python files directly including the migrations ones

Fixed from:
 - https://github.com/OCA/pylint-odoo/pull/348
